.github/workflows/main.yml

```diff
@@ -4,7 +4,6 @@ on:
   pull_request:
     branches:
       - master
-      - 2.*-production
```

Comment:
Does this mean we cannot have the check on the 2.3-production branch? Or are we fixing that in a separate PR?

Reply:
This is a maintenance change that considers that we do not expect any merges to `2.*-production` in the master "tree".